metadata/transportation/road_centerlines

```diff
@@ -8,42 +8,37 @@ a6cf0b81-a99e-4ec3-a9b7-7372ae6a6cbb
 
-This data model is meant to suggest 'best practices'. It is not the intent that all stewards will change their current data management process to match this standard, but rather that stewards will evaluate their current practices to see how well their data 'translates' to this schema for data sharing and Next Generation 911 preparedness.
-
-<!--- Is it common practice for data stewards to change their data to fit ours?  -->
+As part of UGRC's ongoing effort to standardize schema and improve data shareability, this model has been designed to suggest 'best practices' for stewards. We suggest using this model to estimate how well a given dataset 'translates' to this schema and [Next Generation 911 preparedness](https://www.911.gov/issues/ng911/).
 
 ### What does the dataset represent?
 
 <!--- How did we decide to delineate the individual polylines? I noticed that for many roads there is more than one polyline segment for one named road. -->
 
 ### How was the dataset created?
 
-The Utah Roads dataset utilizes a data model based on [Next-Generation 911 standards](https://www.911.gov/issues/ng911/) and the Federal Highway Administration's All Roads Network Of Linear-referenced Data ([ARNOLD](https://highways.dot.gov/safety/data-analysis-tools/rsdp/rsdp-tools/arnold-reference-manual)) reporting requirements for state DOTs. UGRC adopted this data model on September 13th, 2017. The statewide roads dataset is maintained by UGRC in partnership with local government, UDOT, and the 911 community.
+The Utah Roads dataset utilizes a data model based on Next-Generation 911 (NG911) standards and the Federal Highway Administration's ([ARNOLD](https://highways.dot.gov/safety/data-analysis-tools/rsdp/rsdp-tools/arnold-reference-manual)) reporting requirements for state DOTs. UGRC adopted this data model on September 13, 2017. This statewide roads dataset is maintained by UGRC in partnership with local government, UDOT, and NG911 officials. 
 
 ### How reliable and accurate is the dataset?
 
-We are currently working with US Forest Service to improve the Forest Service roads in this dataset, however, for the most current and complete set of USFS roads, please visit the
-[FSGeodata Clearinghouse](https://data.fs.usda.gov/geodata/edw/datasets.php?dsetCategory=transportation).
-
-Please direct questions about this layer to Greg Bunce from UGRC at [gbunce@utah.gov] or by calling 801-349-0039.
+This is the most accurate representation of statewide roads. Please direct questions about this layer to Greg Bunce from UGRC at [gbunce@utah.gov] or by calling 801-349-0039.
```

Comment:
**suggestion**: let's not put names and contact information in the metadata as it becomes technical debt we need to keep current.

Reply:
I wonder if we draw the line and say we don't put contact info in the metadata but it is accessible via our website is that good enough?

Reply:
yeah, i don't see the need for contact info in the metadata. getting it via the website is good for now.

Reply:
I'll go ahead and remove the contact information from this layer. I've noticed that many of the layers in the SGID include contact information--should I remove contact info from those as well? IE, is this the general guideline moving forward?

Reply:
The first project i worked on when i started at agrc was to remove someones contact information from every dataset in the sgid. My vote would be to use a generic contact for UGRC. Then whenever someone retires and it needs to be changed, we don't create more work for ourselves.

Reply:
i fully support that way of thinking. that's why we have steve.

Reply:
Agree! I'll start removing contact information from all the metadata.